testdata/p4_16_samples_outputs/issue1001-1-bmv2-first.p4

```diff
@@ -10,7 +10,7 @@ enum bit<8> PreservedFieldList {
 }
 
 struct Meta {
-    @field_list(PreservedFieldList.Field)
+    @field_list(PreservedFieldList.Field) 
```

Comment:
Here as well, this change is unrelated

Reply:
I will address this while raising review in open source